html/elements/script.json

```diff
@@ -602,10 +605,12 @@
                   "version_added": false
                 },
                 "opera": {
-                  "version_added": "48"
+                  "version_added": "48",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "opera_android": {
-                  "version_added": "45"
+                  "version_added": "45",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "safari": {
```

Comment:
From testing, Safari was the surprise: it appears that modules are not supported in XHTML at all. Only the `text/javascript` script ran. I expect that this is also true for Safari for iOS. Perhaps we could add a note like `Module scripts do not load when the page is served as XHTML (<code>application/xhtml+xml</code>).` to both browsers

Reply:
Actually, this was no surprise to me, thanks for confirming!
I've had someone say one of my sites didn't work before, and it happened to be that they used Mac & Safari.